### ISSUE TITLE
More uses for testosterone

### ISSUE THREAD

**Issue by Dummmmmmmmmy** [MayaqqDev/Estrogen#212]
I'm a guy and want to be able to have a dash as well without having boobs, I know if I use a estrogen pill or patch and two gender change potion the boobs go away, but the boobs still stay just sunk into the chest, so they peek out a little when I move and it looks bad.

My suggestion is to add a testosterone pill or gel/patch to be a guy while still getting the dash

**Comment by MayaqqDev** [comment]
There will be more uses for testosterone in later updates, though they will have unique mechanics compared to estrogen.

**Comment by cxzlw** [comment]
> I'm a guy and want to be able to have a dash as well without having boobs, I know if I use a estrogen pill or patch and two gender change potion the boobs go away, but the boobs still stay just sunk into the chest, so they peek out a little when I move and it looks bad.
> 
> My suggestion is to add a testosterone pill or gel/patch to be a guy while still getting the dash

Negative. Hormones do not work like this. You can't reduce boobs size with hormones.

**Comment by SuperficialCake** [comment]
> > I'm a guy and want to be able to have a dash as well without having boobs, I know if I use a estrogen pill or patch and two gender change potion the boobs go away, but the boobs still stay just sunk into the chest, so they peek out a little when I move and it looks bad.
> > My suggestion is to add a testosterone pill or gel/patch to be a guy while still getting the dash
> 
> Negative. Hormones do not work like this. You can't reduce boobs size with hormones.

This is true that hormones don't work this way, but maybe Maya could add a binder to flatten the chest for those who don't want breasts

**Comment by MayaqqDev** [comment]
You can also just disable em in the config, boobs are not an indicator of gender. Sure they sre interlinked but yeah each person has a personal toggle inside their configs to disable them just for themselves

**Comment by SuperficialCake** [comment]
Honestly had no idea about the toggle, but i also haven’t looked much into the config lol

But anyways, if there’s a toggle then there’s no need for the binder
